source/includes/_responses.md

```diff
@@ -82,6 +82,43 @@ curl -H "Authorization: Bearer myaccesstoken" "https://api.wootric.com/v1/respon
     }
 ]
 ```
+```shell
+$TIMESTAMP = Time.utc(2016, 12, 25).to_i
```

Comment:
I think you're mixing Ruby and Shell together.

Reply:
For Bash, this might work...
```bash
 date -jf "%Y-%m-%d" "2016-12-25" "+%s"
```

Reply:
Ah! makes total sense, thanks for pointing it out @hachibu. I have now updated it to `TIMESTAMP=$(date --date="2016-12-25" "+%s")` what do you think?

Reply:
Looks good